students/synowiec_krzysztof/lesson_07_string_datetime/pythonFile.py

```diff
@@ -0,0 +1,19 @@
+import sys
```

Comment:
not sure which task this should solve. maybe was committed by mistake?

Reply:
Yes, that was mistake. It is my script to create empty file with main function. I copy name from snakify or scope and it conver it name to small leters with _. :)

Reply:
hah, understood :)